.github/workflows/playwright.yml

```
@@ -1,48 +1,83 @@
+
+      - name: List snapshot images from public folder
+        id: list_images
+        run: |
+          if [ -d "public" ]; then
+            echo "FILES=$(ls public/*.png | xargs -n1 basename | tr '\n' ',' | sed 's/,$//')" >> $GITHUB_OUTPUT
+          else
+            echo "FILES=" >> $GITHUB_OUTPUT
+          fi
+
+      - name: Commit snapshot images to PR branch
         run: |
```

actionlint (1.7.7)

"github.head_ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions for more details

(expression)